csm/conf/configure.py

```diff
@@ -166,15 +167,19 @@ def _logrotate(self):
             err_msg = f"logrotate failed. {const.LOGROTATE_DIR_DEST} dir missing."
             Log.error(err_msg)
             raise CsmSetupError(err_msg)
-
-    def _configure_uds_keys(self):
-        Log.info("Configuring UDS keys")
+    
+    def _fetch_management_ip(self):
         cluster_id = Conf.get(const.CONSUMER_INDEX, self.conf_store_keys[const.KEY_CLUSTER_ID])
         virtual_host_key = f"{const.CLUSTER}>{cluster_id}>{const.NETWORK}>{const.MANAGEMENT}>{const.VIRTUAL_HOST}"
         self._validate_conf_store_keys(const.CONSUMER_INDEX,[virtual_host_key])
         virtual_host = Conf.get(const.CONSUMER_INDEX, virtual_host_key)
+        return virtual_host
```

Comment:
Add managment  ip connectivity  validation check

Reply:
Management IP Validation is note required